packages/near-fast-auth-signer/src/components/AddDevice/AddDevice.tsx

```diff
@@ -248,15 +251,15 @@ function SignInPage() {
               const parsedUrl = new URL(success_url || window.location.origin + (basePath ? `/${basePath}` : ''));
               parsedUrl.searchParams.set('account_id', (window as any).fastAuthController.getAccountId());
               parsedUrl.searchParams.set('public_key', public_key);
-              parsedUrl.searchParams.set('all_keys', [public_key, publicKeyFak].join(','));
+              parsedUrl.searchParams.set('all_keys', [public_key, publicKeyFak, recoveryPK].join(','));
```

Comment:
I see this line is duplicated in 215, can we avoid duplicating it and re-use the parsedUrl defined above?